.changeset/happy-tips-burn.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/starlight': minor
+---
+
+Add the `@astrojs/sitemap` and `@astrojs/mdx` integrations only if they are not detected in the Astro configuration.
+
+⚠️ **BREAKING CHANGE** The minimum supported version of Astro is now v3.2.0. Make sure you update Astro at the same time as updating Starlight.
```

Comment:
> This is a breaking change as it requires Astro v3.2.0. Should we link to [Upgrade Astro steps](https://docs.astro.build/en/guides/upgrade-to/v3/#upgrade-astro) on the Astro v3 upgrade guide from the changeset like we did for the Astro v3.0.0 upgrade or is there a more generic page regarding Astro upgrades?

We don’t have generic guidance — although perhaps we should. One option could just be to include that guidance directly here?

````suggestion
⚠️ **BREAKING CHANGE** The minimum supported version of Astro is now v3.2.0. Make sure you update Astro at the same time as updating Starlight:

```sh
npm install astro@latest
```
````

Reply:
Great idea, very clever, did not think of that at all 👍